### ISSUE TITLE
more detailed explanation for NVDA and using with brltty 

### ISSUE THREAD

**Issue by joshknnd1982** [nvaccess/nvda#10033]
### Additional context
I tried connecting my orbit reader to my RCA cambio tablet running windows 10 home 32bit with NVDA and brltty because brltty has a Japanese braille table I want to use. The manual is not very clear on exactly how to use brltty with windows10, though. I installed brltty, installed the driver when it asked, and connected the orbit reader in USB HID mode then set the brltty configuration program to autodetect, USB and bluetooth, so it would just autodetect the display. Then set NVDA to brltty and restartet and got no braille output at all. I tried running some sort of brlapi.exe program in the brltty folder and it said I need python3. I have the latest brltty for windows. I simply want to use the Japanese brltty table to read Japanese braille. brltty has japanese braille wile liblouis does not and I am not sure if the two tables are compatible between brltty and liblouis. Please improve the manual so I can get braille output and switch among brltty tables in windows10 with NVDA.

**Comment by nishimotz** [comment]
I think brltty only supports Japanese phonetic characters.
#2736 is related issue.

**Comment by leonardder** [comment]
I don't think we support nor can support BRLTTY in such a way that it uses its own braille tables for output. BRLTTY support is there to be able to use NVDA with braille displays that aren't support natively by NVDA itself. NVDA is responsible for braille output translation.
